lib/Component/VCard.php

```diff
@@ -357,7 +357,7 @@ function getValidationRules() {
 
             // FN is commented out, because it's already handled by the
             // validate function, which may also try to repair it.
-            // 'FN'           => '+',
+            // 'FN'           => '1',
```

Comment:
Nope it's wrong. http://tools.ietf.org/html/rfc6350#section-6.2.1 says:

    Cardinality:  1*

Which means 'at least one, but can be more.'